README.md

```diff
@@ -1,50 +1,42 @@
-# Plugin used to get analytics from deployments for software platform (gen1) 
+# Airflow Plugin #
 
-## in plugins directory you can find file "astro-analytics"
-Which finds a db session using how telescope does it and then has a function to query the db directly for the information we want.   
-Airflow allows you to make plugins by simply dropping them in the plugins repo and using `AirflowPlugin` class with all the required fields:  
-https://airflow.apache.org/docs/apache-airflow/stable/plugins.html 
+This is an [Airflow plugin](https://airflow.apache.org/docs/apache-airflow/stable/plugins.html) that directly queries the Airflow metadata database and returns certain analytics at an HTTP endpoint.
```

Comment:
```suggestion
This is an [Airflow plugin](https://airflow.apache.org/docs/apache-airflow/stable/plugins.html) that queries the Airflow metadata database and returns certain analytics at an HTTP endpoint.
```

Reply:
Can you explain why you don't like that word?

I included it to indicate that it's reading straight from the database, not from another API endpoint or anything like that.

Reply:
well, it's just a suggestion.  feel free to ignore.

but to me when you say "query the database" you mean... query the database and not e.g. "hit the API" etc... so it seemed that directly didn't mean anything there. but i understand what you're saying 🤷 take it or leave it